modules/dnn/src/layers/layers_common.cpp

```diff
@@ -150,18 +150,12 @@ void getPoolingKernelParams(const LayerParams &params, std::vector<size_t>& kern
 {
     bool is_global = params.get<bool>("global_pooling", false);
     globalPooling = std::vector<bool>(3, is_global);
-    if (params.has("global_d"))
-    {
-        globalPooling[0] = params.get<bool>("global_d");
-    }
-    else if (params.has("global_h"))
-    {
-        globalPooling[1] = params.get<bool>("global_h");
-    }
-    else if (params.has("global_w"))
-    {
-        globalPooling[2] = params.get<bool>("global_w");
-    }
+    if (params.has("global_pooling_d"))
+        globalPooling[0] = params.get<bool>("global_pooling_d");
+    else if (params.has("global_pooling_h"))
+        globalPooling[1] = params.get<bool>("global_pooling_h");
+    else if (params.has("global_pooling_w"))
+        globalPooling[2] = params.get<bool>("global_pooling_w");
```

Comment:
What if we have two of the flags simultaneously? I think we need to have just `if`s here.

Reply:
Thanks! 🙈